ParisBNF/abb/BNFabb94.xml

```diff
@@ -0,0 +1,324 @@
+                                 <material key="parchment">Parchment</material>
+                                         </support>
+                                      <extent>
+                                          <measure unit="leaf">62</measure>
+                                          <measure unit="leaf" type="blank"/>
+                                          <locus target="#1r #1v #32v #33r #33v #55v #56r #63r #63v"/>
+                                          <dimensions type="outer" unit="mm">
+                                            <height>210</height>
+                                            <width>150</width>
+                                          </dimensions>
+                                      </extent>
+
+                                      <foliation>European foliation in the upper right corner of the folia.
+                                        </foliation>
+
+<collation>
+<list>
+  <item xml:id="q1" n="A"><dim unit="leaf">1</dim><locus from="ir" to="iv"></locus></item>
+  <item xml:id="q2" n="1"><dim unit="leaf">9</dim><locus from="1r" to="9v"></locus></item>
+  <item xml:id="q3" n="2"><dim unit="leaf">8</dim><locus from="10r" to="17v"></locus></item>
+  <item xml:id="q4" n="3"><dim unit="leaf">4</dim><locus from="18r" to="21r"></locus></item>
+  <item xml:id="q5" n="4"><dim unit="leaf">8</dim><locus from="22r" to="29v"></locus></item>
+  <item xml:id="q6" n="5"><dim unit="leaf">4</dim><locus from="30r" to="33v"></locus></item>
+  <item xml:id="q7" n="6"><dim unit="leaf">8</dim><locus from="34r" to="41v"></locus></item>
+  <item xml:id="q8" n="7"><dim unit="leaf">7</dim><locus from="42r" to="48v"></locus></item>
+  <item xml:id="q9" n="8"><dim unit="leaf">7</dim><locus from="49r" to="55v"></locus></item>
+  <item xml:id="q10" n="9"><dim unit="leaf">7</dim><locus from="56r" to="63v"></locus></item>
+</list>
+</collation>
+
+                                      <condition key="good"></condition>
+
+                                      </supportDesc>
+
+
+                              </objectDesc>
+
+<decoDesc>
+  <decoNote type="ornamentation" xml:id="d1">
+        <locus target="#2r #22r #34r #42r #47r #53r"/>
+        <desc>Simple ornamental bands.</desc>
+    </decoNote>
+
+    <decoNote type="ornamentation" xml:id="d2">
+          <locus target="#57r"/>
+          <desc>Ḥarag.</desc>
+      </decoNote>
+    
+    <decoNote type="ornamentation" xml:id="d3">
+        <locus target="#57v #58r"/>
+        <desc>Crux ansata.</desc>
+    </decoNote>
+
+</decoDesc>
+
+
+                              <additions>
+                                <list>
+                                <item xml:id="a1">
+                                   <desc type="OwnershipNote"/>
+                                   <locus target="#1v"/>
+                                   <q xml:lang="gez">ዝመጽሐፍ፡ ገድሎሙ፡ ለቅዱሳን፡ አባ፡ አቡናፍር፡ ወአባ፡ ኪሮስ፡ ወመልከ፡ ጼዴቅ፡ ወድርሳነ፡ አፍኒን፡ ዘመካነ፡ <placeName ref="INS0329DE">ዳጋ።</placeName></q>
```

Comment:
you do not want to create pers ids for these people? If you do not want to, I will do it myself, no worries!